SingleSource/UnitTests/AArch64/acle-fmv-features.c

```diff
@@ -196,7 +196,14 @@ CHECK(ssbs2, {
 CHECK(bti, {
     // The only test for this requires reading a register that is only
     // accessible to EL1.
-    #if defined(__APPLE__)
+    #ifdef __linux__
```

Comment:
Could use something like this as a continuation of the comment above:
```suggestion
    #ifdef __linux__
        // On linux, the kernel intercepts the trap, and emulates it for EL0 processes.
```